packages/keymanager-server/test/unit/impl.test.ts

```diff
@@ -0,0 +1,138 @@
+import {assert} from "chai";
+import sinon from "sinon";
+import {KeymanagerApi} from "../../src";
+import {Interchange, SlashingProtection, Validator, ValidatorStore} from "@chainsafe/lodestar-validator";
+import {Root} from "@chainsafe/lodestar-types";
+import Sinon from "sinon";
+import fs from "node:fs";
+import {Keystore} from "@chainsafe/bls-keystore";
+import lockfile from "lockfile";
+import {testLogger} from "@chainsafe/lodestar-validator/test/utils/logger";
+
+describe("keymanager", () => {
+  let validatorSub: Sinon.SinonStubbedInstance<Validator>;
+  let validatorStoreSub: Sinon.SinonStubbedInstance<ValidatorStore>;
+  let slashingProtectionStub: Sinon.SinonStubbedInstance<SlashingProtection>;
+  let genesisValidatorRootStub: Sinon.SinonStubbedInstance<Uint8Array | Root>;
+  let keystore: Sinon.SinonStubbedInstance<Keystore>;
+  const interchange: Interchange = ("abc" as unknown) as Interchange;
+  const PASSWORD = "passfortesting";
+  // eslint-disable-next-line
+  const keyStoreStr = "{\"crypto\": {\"kdf\": {\"function\": \"scrypt\", \"params\": {\"dklen\": 32, \"n\": 262144, \"r\": 8, \"p\": 1, \"salt\": \"34d24f0a6f85b7b55d5ccc54efd0ba2955472a39a72d55e1c71fb770717639de\"}, \"message\": \"\"}, \"checksum\": {\"function\": \"sha256\", \"params\": {}, \"message\": \"c043032c9c50ebcaab5ec6edccad095d223ba5e7be40b2a39b8931ab931585a3\"}, \"cipher\": {\"function\": \"aes-128-ctr\", \"params\": {\"iv\": \"53e5deb6df661b998140ca59de04bd69\"}, \"message\": \"c3fa507c9fdb0bf14983a09175210b6d650d89451bb5a39f368e2f0421db0b14\"}}, \"description\": \"\", \"pubkey\": \"8cd1ea594e011cbdae67c583206aef8661f74a800082079e4edf96b86eb631fff236fcf6b87b57153c26d76c65bc7970\", \"path\": \"m/12381/3600/0/0/0\", \"uuid\": \"a7fa0c0f-edd6-4640-b46d-872db3696a36\", \"version\": 4}";
+
+  beforeEach(() => {
+    validatorSub = sinon.createStubInstance(Validator);
+    validatorStoreSub = sinon.createStubInstance(ValidatorStore);
+    slashingProtectionStub = sinon.createStubInstance(SlashingProtection);
+    genesisValidatorRootStub = sinon.createStubInstance(Uint8Array);
+    keystore = sinon.createStubInstance(Keystore);
+  });
+
+  afterEach(() => {
+    sinon.restore();
+  });
+
+  describe("Keymanager / importKeystores", () => {
```

Comment:
Switching to real data would probably be still cumbersome. Since I later added code to easily perform e2e testing, I'll remove this test completely to have the main flow covered by the e2e test